src/frontend/use-replicache.ts

```diff
@@ -15,38 +15,41 @@ export function useReplicache<M extends MutatorDefs>(
   // Currently Replicache is only supported on the client-side. The useEffect()
   // here is a common hack to prevent next from running the code server-side.
   useEffect(() => {
-    (async () => {
```

Comment:
This `async` wrapper was actually no longer needed. I guess at some point we had some `await`s in here but we don't anymore.